From: Julian Andres Klode <julian.klode@canonical.com>
Date: Thu, 25 Jan 2018 12:02:55 +0100
Subject: 11-dm-mpath: fix DM_UDEV_RULES_VSN-check

Fix 11-dm-mpath.rules to check for DM_UDEV_RULES, not DM_UDEV_RULES_VSN

We patch that in lvm2 to be named differently for who knows why.

Bug-Debian: https://bugs.debian.org/886938
Forwarded: not-needed
Origin: vendor
---
 multipath/11-dm-mpath.rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/multipath/11-dm-mpath.rules b/multipath/11-dm-mpath.rules
index 7fa4aab..51d01c6 100644
--- a/multipath/11-dm-mpath.rules
+++ b/multipath/11-dm-mpath.rules
@@ -1,5 +1,5 @@
 ACTION!="add|change", GOTO="mpath_end"
-ENV{DM_UDEV_RULES_VSN}!="?*", GOTO="mpath_end"
+ENV{DM_UDEV_RULES}!="?*", GOTO="mpath_end"
 ENV{DM_UUID}!="mpath-?*", GOTO="mpath_end"
 
 IMPORT{db}="DM_DISABLE_OTHER_RULES_FLAG_OLD"
